lib/logstash/environment.rb

```diff
@@ -114,6 +95,42 @@ def bundler_setup!(options = {})
       ::Bundler.setup
     end
 
+    def runtime_jars_root(dir_name, package)
```

Comment:
In the plugins we do:

```
ROOT_DIR   = File.expand_path(File.join(File.dirname(__FILE__), ".."))
LogStash::Environment.load_runtime_jars! File.join(ROOT_DIR, "vendor")
```

But that's a lot of unecessary code we need on each plugin. At most, we need `LogStash::Environment.load_runtime_jars! File.dirname(__FILE__)` and even that we could somehow find by ourselves.

If we reduce this to just `LogStash::Environment.load_runtime_jars!`, we no longer need this extra file and can just include the require call in the plugin code. thoughts?